### PR TITLE
add gooddeed model

### PR DESCRIPTION
**Pull request by michellewehr**

### CODE REVIEW

File: server/models/Event.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+         ],
+        location: {
+            type: String,
+            required: true,
+            trim: true
+          },
+        description: {
+            type: String, 
+            required: true,
+            maxlength: 280
```
Comment (brentocracy) [inline server/models/Event.js:31]: Maybe we can have a larger `maxLength`? Up for debate, I think doubling it to 560 might allow for better descriptions.

File: server/models/Event.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            type: String,
+            required: true
+        },
+        endTime: {
+            type: String, 
+            required: true
+        },
+        url: {
+            type: String, 
+            required: false
```
Comment (brentocracy) [inline server/models/Event.js:47]: Do we need `required: false`? I think if we don't include the attribute at all, it defaults to that.

File: server/models/GoodDeed.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        },
+        deedText: {
+            type: String,
+            required: true,
+            trim: true
+        },
+        createdAt: {
+            type: Date,
+            default: Date.now,
+            get: timestamp => dateFormat(timestamp)
```
Comment (brentocracy) [inline server/models/GoodDeed.js:25]: I think there might be a shorter/more simple date format function we can use, but if this one works, it's fine with me.

**Review by brentocracy — APPROVED** [review]
Looks good to me. We can test our data with seeds once all of the models are set up.

File: server/models/Event.js (hunk 1 above)
Comment (DaveToth77) [reply server/models/Event.js:31]: Definitely wouldn't hurt the app at this stage to up the character count

Comment (brentocracy) [reply server/models/Event.js:31]: Cool, maybe a small change for the next commit.

**Review by DaveToth77 — COMMENTED** [review]
looks good

**Review by DaveToth77 — APPROVED** [review]

**Pull request merged**